content/id/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,161 @@
+
+Opsi `percentageOfNodesToScore` menerima semua angka numerik antara 0 dan 100.
+Angka 0 adalah angka khusus yang menandakan bahwa kube-scheduler harus menggunakan
+nilai _default_.
+Jika kamu mengatur `percentageOfNodesToScore` dengan angka di atas 100, kube-scheduler
+akan membulatkan ke bawah menjadi 100.
+
+Untuk mengubah angka, sunting berkas konfigurasi kube-scheduler (berkas `/etc/kubernetes/config/kube-scheduler.yaml`),
+lalu _restart_ ulang penjadwal.
+
+Setelah kamu selesai menyunting, jalankan perintah
+```bash
+kubectl get componentstatuses
+```
+untuk verifikasi komponen kube-scheduler berjalan dengan baik (_healthy_). Keluarannya kira-kira seperti ini:
+```
+NAME                 STATUS    MESSAGE             ERROR
+controller-manager   Healthy   ok
+scheduler            Healthy   ok
+...
+```
+
+## _Threshold_ penilaian Node {#persentase-penilaian-node}
+
+Untuk meningkatan performa penjadwalan, kube-scheduler dapat berhenti mencari
+Node-Node yang layak saat sudah berhasil menemukannya. Pada klaster berskala besar,
+ini menghemat waktu dibandingkan dengan pendekatan awam yang mengecek setiap Node.
+
+Kamu bisa mengatur _threshold_ untuk menentukan berapa banyak jumlah Node minimal yang dibutuhkan, sebagai
+persentase bagian dari seluruh Node di dalam klaster kamu. kube-scheduler akan mengubah ini menjadi
+bilangan bulat berisi jumlah Node. Saat penjadwalan, jika kube-scheduler mengidentifikasi
+Node-Node layak yang diperlukan telah melewati jumlah persentase yang diatur, maka kube-scheduler
+akan berhenti mencari Node-Node layak dan lanjut ke [fase penilaian] (/docs/concepts/scheduling/kube-scheduler/#kube-scheduler-implementation).
+
+[Bagaimana penjadwal mengecek Node](#bagaimana-penjadwal-mengecek-node) menjelaskan proses ini secara detail.
+
+### _Threshold default_
+
+Jika kamu tidak mengatur sebuah _threshold_, maka Kubernetes akan melakukan
+perhitungan menggunakan pendekatan linier, yaitu 50% untuk klaster dengan 100 Node,
+serta 10% untuk klaster dengan 5000 Node.
+
+Artinya, kube-scheduler selalu menilai paling tidak 5% dari klaster kamu, terlepas dari
+seberapa besar klasternya, kecuali kamu secara eksplisit mengatur `percentageOfNodesToScore`
+untuk lebih kecil dari 5.
+
+Jika kamu ingin penjadwal untuk memasukkan seluruh Node di dalam klaster ke dalam penilaian, 
+maka atur `percentageOfNodesToScore` menjadi 100.
+
+## Contoh
+
+Contoh konfigurasi di bawah ini mengatur `percentageOfNodesToScore` menjadi 50%.
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
+algorithmSource:
+  provider: DefaultProvider
+
+...
+
+percentageOfNodesToScore: 50
+```
+
+
+##  Menyetel percentageOfNodesToScore
+
+`percentageOfNodesToScore` merupakan angka 1 sampai 100 dengan
+_default_ yang dihitung berdasarkan ukuran klaster. Kamu juga bisa meng-_hardcode_
+menjadi angka minimal 50 Node.
+
+{{< note >}}Pada klaster dengan kurang dari 50 Node-Node layak, penjadwal masih
```

Comment:
```suggestion
{{< note >}}Pada klaster dengan kurang dari 50 Node layak, penjadwal masih
```

Reply:
Fixed in https://github.com/kubernetes/website/pull/20260/commits/3cc163e65705127d1fcf16feba108179f9bdacc2